configure.ac

```diff
@@ -375,6 +376,27 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 )
 CXXFLAGS="$TEMP_CXXFLAGS"
 
+case $host in
+  *mingw*)
```

Comment:
Why is this disallowed on mingw? Add a comment...

Reply:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=79127

Reply:
Shouldn't your below test cleanly fail in that scenario?

Reply:
Most servers are linux and support avx512?

Reply:
That's irrelevant...

Reply:
How do I do？ update ci mingw version?

Reply:
You shouldn't need to.